### PR TITLE
[I18N] Added german translations

### PR DESCRIPTION
**Pull request by Jan-NiklasB**
@RocketChat/android

### CODE REVIEW

**Comment by Jan-NiklasB** [comment]
Faild checks don't seem to are directly related to the changes made...

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -159,18 +159,15 @@
...
+    <string name="msg_view_more">Zeige mehr</string>
+    <string name="msg_view_less">Zeige weniger</string>
+    <string name="msg_permalink_copied">Permanenter Link kopiert</string>
 
     <!-- Preferences messages -->
-    <string name="msg_analytics_tracking">Analytics tracking</string> <!-- TODO Add translation -->
-    <string name="msg_send_analytics_tracking">Send anonymous statics to help improving this app</string> <!-- TODO Add translation -->
-    <string name="msg_do_not_send_analytics_tracking">Do not send anonymous statics to help improving this app</string> <!-- TODO Add translation -->
-    <string name="msg_not_applicable_since_it_is_a_foss_version">Not applicable since it is a FOSS version</string> <!-- TODO Add translation -->
+    <string name="msg_analytics_tracking">Google Analytics verfolgung</string>
```
Comment (philipbrito) [inline app/src/main/res/values-de/strings.xml]: Could you remove the `Google` here please?

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -132,18 +132,18 @@
...
-    <string name="msg_continue_with_linkedin">Continue with <b>Linkedin</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_gitlab">Continue with <b>GitLab</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_wordpress">Continue with <b>WordPress</b></string> <!-- TODO Add translation -->
-    <string name="msg_two_factor_authentication">Two-factor Authentication</string> <!-- TODO Add translation -->
-    <string name="msg__your_2fa_code">What’s your 2FA code?</string> <!-- TODO Add translation -->
+    <string name="msg_welcome_to_rocket_chat">Wilkommen bei Rocket.Chat</string>
+    <string name="msg_team_communication">Open Source Kommunikation</string>
+    <string name="msg_login_with_email">Mit <b>e-mail</b> einloggen</string>
+    <string name="msg_create_account">Einen Account erstellen</string>
+    <string name="msg_continue_with_facebook">Mit <b>Facebook</b> einloggen</string>
```
Comment (philipbrito) [inline app/src/main/res/values-de/strings.xml:163]: Is `Mit` the translate to `Login` in English here? If so, could you translate it to the `Continue` in English?

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by philipbrito** [comment]
@Jan-NiklasB It has failed because you send the PR to the `master` branch. It should be pointed to `develop`. I have changed it already. Could you fix the conflicts, please?

**Comment by Jan-NiklasB** [comment]
Sure, I will do it as soon as I'm at home.

For the "mit" which means "with" I don't exactly know where this sentence is used so I translated it in a way that made the Most sence.
The Translation back to english would be " Login with FB" but in german the word by word translation doesn't makes much sence (which would be "Mit FB fortfahren") since continue or fortfahren has multiple meanings in german and would be Not very precise.

So you must decide how it should be translated... ;)

**Comment by philipbrito** [comment]
Perfect @Jan-NiklasB! Thanks for explaining (and teaching a little bit of German here). 
Taking advantage: Is `Analytics verfolgung` enough/good in German? If so let's only remove the `Google` word.

Thank you.

**Comment by philipbrito** [comment]
We'll be grateful if you fix the conflict here too. Thanks!

**Comment by Jan-NiklasB** [comment]
@filipedelimabrito You're welcome :D 
I think for people who've no lingual relation to german like e.g. dutch or swiss german it's really hard to learn.
Especially the grammar xD (german grammar is sometimes very difficult, even for germans).

I changed the analytics string to "Daten für Analysezwecke" what means "data for analyzing purposes", I think that should be presice enougth for everyone.

**Review by philipbrito — APPROVED** [review]
Thanks a lot @Jan-NiklasB 💯

**Pull request merged**